packages/perseus/src/widgets/interactive-graphs/mafs-locked-layer.tsx

```diff
@@ -0,0 +1,41 @@
+import {UnreachableCaseError} from "@khanacademy/wonder-stuff-core";
+import {Point} from "mafs";
+import * as React from "react";
+
+import type {LockedFigure} from "../../perseus-types";
+
+type Props = {
+    lockedFigures: Array<LockedFigure>;
```

Comment:
In most places in Perseus we type array props as `ReadonlyArray<>`. If you update that here, I think that'll remove the need to make adjustments to `perseus-types.ts`.

Reply:
Ah! That would do it.